src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java

```diff
@@ -218,6 +222,20 @@ public Builder setRequireAuthentication() {
       return this;
     }
 
+    /** Enables OpenTelemetry tracing for PGAdapter. */
+    public Builder setEnableOpenTelemetry() {
```

Comment:
We are planning to make the Otel metrics in the client library as opt-in by default. So i think we should make this also as opt-out.

Reply:
What exactly do you mean by `opt-in by default`. Does that mean that you will enable it by default? If so, what exactly do you mean with enabled by default?
1. Does it mean 'we enable OpenTelemetry by default, but we don't by default add any exporters'? If so, I would argue that this is still the same as the feature being an opt-in, as you still need to add an exporter before you actually collect any metrics.
2. Or does it mean that we enable both OpenTelemetry and add an exporter (e.g. Google Cloud Tracer) by default? That would mean that we by default enable a feature that could incur additional costs for the customer. Is that intentional?

Reply:
I meant the latter. But I realize now that itsn not possible for traces as we cannot specify a namespace.  So lets leave this as-is. Thank you!